frontend/src/components/organisms/AddingPlayerForm/AddingPlayerForm.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { Button, Grid, TextField, Typography } from "@material-ui/core";
+import { VerticalSpace } from "../../atoms/Shared/VerticalSpace";
+import { useFormik } from "formik";
+import * as yup from "yup";
+import { UserProfileRestApi } from "../../../restapi/players-profiles";
+import { EntityIdGenerator } from "../../idGenerator/EntityIdGenerator";
+
+const validationSchema = yup.object({
+  name: yup.string().required("To pole jest wymagane."),
+  surname: yup.string().required("To pole jest wymagane."),
+  email: yup
+    .string()
+    .email("Niepoprawny adres e-mail.")
+    .required("To pole jest wymagane."),
+  phone: yup
+    .string()
+    .min(9, "Wprowadź przynajmniej 9 znaków.")
+    .required("To pole jest wymagane."),
+});
+
+function AddingPlayerForm() {
+  //TODO dead code below might be used for TODO below
```

Comment:
Tutaj moze byc w props cos w stylu onPlayerAdded. Zeby nie wpakowywac turniejow w ten komponent.

Reply:
czemu turniejów? Z path pobiorę sobie tylko id turnieju

Reply:
No wlasnie wtedy masz to powiazane - ze AddingPlayer wie o turnieju.
Proponuje. Dodajesz do props onPlayerAdded()
I teraz tak w onSubmit: zostawiasz to co jest i czyscisz formularz i wyowłujesz props.onSubmit()

A w komponencie wyżej robisz, że onSubmit zamyka drawer/ przeladowuje liste (bedzie z nowym graczem) no i wysyła kolejny request. W zyciu ten reaload :) Wtedy masz 2 rzeczy w jednym.
Dobry design = dobre mozliwosci.